src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -243,11 +243,27 @@ private void OnDeserialized(StreamingContext context)
         }
     }
 
+    [Flags]
+    internal enum StackTraceFormattingOptions
+    {
+        None                                 = 0,
+        ExcludeStackTraceHiddenAttribute     = 1 << 0,
+        ExcludeDispatchBoundaries            = 1 << 1,
+        ExcludeInnerExceptionBoundaries      = 1 << 2,
```

Comment:
Concatenating across inner exceptions feels like another one that we should segregate out of this PR and bring up separately. It doesn't seem unreasonable as something people would want, but I don't feel I've seen much feedback on it either way. I'd rather not lump it with the async stuff where we have seen significant feedback already.